coverage/artifacts/local_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
+
+import (
+	"testing"
+
+	"k8s.io/test-infra/coverage/test"
+)
+
+// generates coverage profile by running go test on target package
+func TestComposeCmdArgs(t *testing.T) {
```

Comment:
Please structure these idiomatically:


```go
func TestComposeCmdArgs(t *testing.T) {
    var testCases = []struct{
        name string
        input, expected []string
    }
    for _, testCase := range testCases {
        // ...
    }
}
```

Use the `name` field to explain what the input is and what the expected output is, like `"name": "absolute path is invalid"`